### ISSUE TITLE
Brill demos fail by calling unknown method batch_test()

### ISSUE THREAD

**Issue by LenRemmerswaal** [nltk/nltk#1032]
I tried to use `nltk.tbl.demo` functions, but many crash on calling `brill_tagger.batch_tag(testing_data)` in function `postag` as there is no `batch_tag()` method.
I fixed this, and another bug (possibly #129) with the following patch.

Versions:
- nltk: 3.0.2
- python 2.7.9

The patch:

```
`diff -r dcebb5dac345 tbl/demo.py
--- a/tbl/demo.py   Tue Jun 30 10:10:43 2015 -0400
+++ b/tbl/demo.py   Thu Jul 02 16:43:14 2015 -0400
@@ -268,7 +268,7 @@
             print("Wrote plot of learning curve to {0}".format(learning_curve_output))
     else:
         print("Tagging the test data")
-        taggedtest = brill_tagger.batch_tag(testing_data)
+        taggedtest = brill_tagger.tag_sents(testing_data)
         if template_stats:
             brill_tagger.print_template_statistics()

@@ -276,20 +276,19 @@
     if error_output is not None:
         with open(error_output, 'w') as f:
             f.write('Errors for Brill Tagger %r\n\n' % serialize_output)
-            for e in error_list(gold_data, taggedtest):
-                f.write(e+'\n')
+            f.write(u'\n'.join(error_list(gold_data, taggedtest)).encode('utf-8') + '\n')
         print("Wrote tagger errors including context to {0}".format(error_output))

     # serializing the tagger to a pickle file and reloading (just to see it works)
     if serialize_output is not None:
-        taggedtest = brill_tagger.batch_tag(testing_data)
+        taggedtest = brill_tagger.tag_sents(testing_data)
         with open(serialize_output, 'w') as print_rules:
             pickle.dump(brill_tagger, print_rules)
         print("Wrote pickled tagger to {0}".format(serialize_output))
         with open(serialize_output, "r") as print_rules:
             brill_tagger_reloaded = pickle.load(print_rules)
         print("Reloaded pickled tagger from {0}".format(serialize_output))
-        taggedtest_reloaded = brill_tagger.batch_tag(testing_data)
+        taggedtest_reloaded = brill_tagger.tag_sents(testing_data)
         if taggedtest == taggedtest_reloaded:
             print("Reloaded tagger tried on test set, results identical")
         else:
```

**Comment by LenRemmerswaal** [comment]
Patch now in pull request #1037

**Comment by stevenbird** [comment]
Closed by #1037.
